django/cantusdb_project/main_app/views/user.py

```diff
@@ -109,6 +110,7 @@ def get_context_data(self, **kwargs):
         page_obj = paginator.get_page(page_number)
 
         context["user_created_sources_page_obj"] = page_obj
+        context["user_is_proofreader"] = user_is_proofreader(self.request.user)
```

Comment:
Looking at `user_detail.html`, I can't see `user_is_proofreader` anywhere. Is it necessary to add it to `context`?

Reply:
It's used in `user_source_list.html`, so it should be added to `context` in this case

Reply:
Ah, okay! Somehow I thought this was part of a different view. Very good!